apps/chatbot/package.json

```diff
@@ -27,15 +27,11 @@
   },
   "devDependencies": {
     "@types/node": "~20.11.30",
-    "@types/prettier": "^2.7.3",
-    "@typescript-eslint/eslint-plugin": "^6.20.0",
-    "@typescript-eslint/parser": "^6.20.0",
-    "eslint": "^8.56.0",
-    "eslint-plugin-import": "^2.29.1",
     "typescript": "^5.3.3"
   },
   "scripts": {
     "start": "tsx src/server.ts",
-    "lint": "eslint ."
+    "lint": "biome check",
+    "fix": "biome check --write"
```

Comment:
Related to the comment on the root `package.json`, should these be passing `.`?

Reply:
Since version 1.8.0:

> Biome now executes commands (lint, format, check and ci) on the working directory by default